interface/python/src/init_MixtureOptions.cpp

```diff
@@ -0,0 +1,864 @@
+		    "Returns the unitless vector of species specific heats at constant pressure \f$ C_{p,i} / R_u \f$."
+		    )
+
+		.def("speciesCpOverR",
+		    [](const Mutation::Mixture &self, const double T)
+		    {
+		        std::vector<double> cp_i(self.nSpecies());
+		        self.speciesCpOverR(T, cp_i.data());
+		        return cp_i;
+		    },
+		    "Returns the unitless vector of species specific heats at constant pressure "
+		    "\f$ C_{p,i} / R_u \f$ given temperature in K."
+		    )
+
+		.def("mixtureFrozenCpMole",
+		    &Mutation::Mixture::mixtureFrozenCpMole,
+		    "Returns the frozen mixture averaged specific heat at constant pressure in J/mol-K."
+		    )
+
+		.def("mixtureFrozenCpMass",
+		    &Mutation::Mixture::mixtureFrozenCpMass,
+		    "Returns the frozen mixture averaged specific heat at constant pressure in J/kg-K."
+		    )
+
+		.def("mixtureEquilibriumCpMole",
+		    &Mutation::Mixture::mixtureEquilibriumCpMole,
+		    "Returns the current equilibrium mixture averaged specific heat at"
+            " constant pressure in J/mol-K. This method assumes that the current state"
+            " of the mixture was already set using the equilibrate() method."
+		    )
+
+		.def("mixtureEquilibriumCpMass",
+		    &Mutation::Mixture::mixtureEquilibriumCpMass,
+		    "Returns the current equilibrium mixture averaged specific heat at"
+            " constant pressure in J/kg-K.  This method assumes that the current state"
+            " of the mixture was already set using the equilibrate() method."
+		    )
+
+		.def("mixtureFrozenCvMole",
+		    &Mutation::Mixture::mixtureFrozenCvMole,
+		    "Returns the mixture averaged specific heat at constant volume in J/mol-K."
+		    )
+
+		.def("mixtureFrozenCvMass",
+		    &Mutation::Mixture::mixtureFrozenCvMass,
+		    "Returns the mixture averaged specific heat at constant volume in J/kg-K."
+		    )
+
+		.def("mixtureEquilibriumCvMass",
+		    &Mutation::Mixture::mixtureEquilibriumCvMass,
+		    "Returns the current equilibrium mixture averaged specific heat at"
+            " constant volume in J/kg-K.  This method assumes that the current state of"
+            " the mixture was already set using the equilibrate() method."
+		    )
+
+		.def("mixtureFrozenGamma",
+		    &Mutation::Mixture::mixtureFrozenGamma,
+		    "Returns the ratio of mixture averaged specific heats which is a unitless quantity."
+		    )
+
+		.def("mixtureEquilibriumGamma",
+		    &Mutation::Mixture::mixtureFrozenGamma,
+		    "Returns the current equilibrium ratio of mixture averaged specific heats which is a unitless quantity."
+		    )
+
+
+		.def("X",
+		    [](const Mutation::Mixture &self)
+		    {
+                std::vector<double> x(self.X(), self.X() + self.nSpecies());
+		        return x;
+		    },
+		    "Returns the current species mole fractions."
+		    )
+
+		.def("Y",
+		    [](const Mutation::Mixture &self)
+		    {
+                std::vector<double> x(self.Y(), self.Y() + self.nSpecies());
```

Comment:
Suggestion for consistent formatting: configure and use clang-format (you'll make your reviewers very happy ;) ).